runners/launch_docker-dev.sh

```diff
@@ -33,27 +34,28 @@ while test $# -gt 0; do
 done
 
 DOCKER_IMAGE="humancompatibleai/imitation:python-req"
-# Specify LOCAL_MNT if you want to mount a local directory to the docker container
-if [[ ${LOCAL_MNT} == "" ]]; then
-  LOCAL_MNT="${HOME}/imitation"
+# Specify IMIT_LOCAL_MNT if you want to mount a local directory to the docker container
+if [[ ${IMIT_LOCAL_MNT} == "" ]]; then
+  IMIT_LOCAL_MNT="${HOME}/imitation"
 fi
 
 # Pass your own mjkey.txt
-if [[ ${MJKEY_MNT} == "" ]]; then
-  MJKEY_MNT="${HOME}/mnt/mjkey.txt"
+if [[ ${IMIT_MJKEY_MNT} == "" ]]; then
```

Comment:
We probably don't need this feature any more since the mjkey.txt is already baked into the docker image.

Reply:
Cool, thanks for the reminder!